test/Common/OSMD/OSMD_Test.ts

```diff
@@ -2,6 +2,14 @@ import chai = require("chai");
 import {OpenSheetMusicDisplay} from "../../../src/OpenSheetMusicDisplay/OpenSheetMusicDisplay";
 import {TestUtils} from "../../Util/TestUtils";
 
+function createOpenSheetMusicDisplay(div: HTMLElement): OpenSheetMusicDisplay {
```

Comment:
Can we move this to TestUtils?

Reply:
yes, good idea. can also be used in Xml_Test.